src/components/SingleArticle/SingleArticle.jsx

```diff
@@ -1,20 +1,31 @@
 import React, { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
-import { getSingleArticle } from "../../api";
-import { Link } from "react-router-dom";
-import CommentCard from "../CommentCard/CommentCard";
+import { getSingleArticle, upVoteArticle } from "../../api";
 import CommentList from "../CommentList/CommentList";
 
 const SingleArticle = () => {
   const [article, setArticle] = useState({});
+  const [votes, setVotes] = useState([])
```

Comment:
As votes is a number, better to initialise it with a number instead of an empty array